calibration/CTDMOG/CGINS-CTDMOG-13643__20170405.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13643,CC_ptempa0,1.576623e+002,
 37-13643,CC_ptempa1,-6.667327e-002,
 37-13643,CC_ptempa2,-1.664682e-007,
-37-13643,CC_wbotc,2.5588e-007,
+37-13643,CC_wbotc,2.558807e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT